README.md

```diff
@@ -20,6 +20,22 @@ However, as devices and software grows more complex, vendors want to perform mor
 In the absence of a standard for third party devices, vendors often have to write and maintain multiple plugins for different runtimes or even directly contribute vendor specific code in the runtime.
 Additionally runtimes don't uniformly expose a plugin system (or even expose a plugin system at all) leading to duplication of the functionality in higher level abstractions (such as Kubernetes device plugins).
 
+## How to configure CDI?
+
+### CRI-O configuration
+
+### Containerd configuration
+
+To enable and configure CDI support in the [containerd runtime](https://github.com/containerd/containerd) 2 configuration options `enable_cdi` and `cdi_spec_dirs` should be set in the `plugins."io.containerd.grpc.v1.cri` section of the containerd configuration file (`/etc/containerd/config.toml` by default):
+
+```
+[plugins."io.containerd.grpc.v1.cri"]
+  enable_cdi = true
+  cdi_spec_dirs = ["/etc/cdi", "/var/run/cdi"]
+```
+
+Remember to restart containerd for any configuration changes to take effect.
+
```

Comment:
Should we call out how the devices are actually specified?

Reply:
Can you elaborate? We have a spec format described [here](https://github.com/container-orchestrated-devices/container-device-interface/blob/master/SPEC.md). Devices should be specified according to the spec.

Reply:
I mean that for `podman` and as mentioned in the readme below we indicate that the devices are passed through the `--device` command line argument. This is not as clear for containerd as this relies on container annotations, for example. If we don't think that that is relevant, then I'm happy to leave this out.

Reply:
Thanks. I'll try to come up with something on this regard.

Reply:
It should be a separate section describing this as the way devices are passed to the CRI runtime are common for CRI-O and Containerd.

Reply:
I think from the user experience point of view it probably would be a good idea to give at least one example, as much self-contained as possible, with step-by-step instructions of how to

- set up CDI for the host
  - enable CDI in a/the runtime
  - create/drop CDI configuration in place
- create a container with an injected CDI device
  - using podman
  - using crictl
- examine/verify the result

Reply:
first step 'set up CDI for the host' is more or less covered by this PR. we still need some explanations on how to 'create/drop CDI configuration in place' may be.

remaining 2 parts 'create a container with an injected CDI device' and 'examine/verify the result' need to be added. I'd propose to do it as a real-life example and put into a separate section in the README.md.